version

```diff
@@ -1 +1 @@
-0.3.58
+0.3.59
```

Comment:
What's this file used for? I'd like to move away from adding an additional commit with a file each time. What value is this file providing us?

Reply:
I could not figure out a way to automate version bumping without registering the current version somehow.

I can try to remove if we remove all bumping helpers.